src/runtime/virtcontainers/qemu.go

```diff
@@ -1304,13 +1304,23 @@ func (q *qemu) hotplugVFIODevice(device *config.VFIODev, op operation) (err erro
 			case config.VFIODeviceNormalType:
 				return q.qmpMonitorCh.qmp.ExecuteVFIODeviceAdd(q.qmpMonitorCh.ctx, devID, device.BDF, device.Bus, romFile)
 			case config.VFIODeviceMediatedType:
+				if utils.IsAPVFIOMediatedDevice(device.SysfsDev) {
+					return q.qmpMonitorCh.qmp.ExecuteAPVFIOMediatedDeviceAdd(q.qmpMonitorCh.ctx, device.SysfsDev)
+				}
 				return q.qmpMonitorCh.qmp.ExecutePCIVFIOMediatedDeviceAdd(q.qmpMonitorCh.ctx, devID, device.SysfsDev, "", device.Bus, romFile)
 			default:
 				return fmt.Errorf("Incorrect VFIO device type found")
 			}
 		}
 
-		addr, bridge, err := q.arch.addDeviceToBridge(devID, types.PCI)
+		var bridgeType types.Type
+		if machinneType == QemuCCWVirtio {
```

Comment:
Is `machinneType` a typo?

Reply:
`machinneType` is the existing variable name. I didn't want to change this in the feature PR and found that should maybe have its own PR.